server/app/assets/javascripts/admin_program_block_edit.ts

```diff
@@ -0,0 +1,48 @@
+import {addEventListenerToElements} from './util'
+import {init as mainInit} from './main'
+
+class AdminProgramBlockEdit {
+  public registerEventListeners() {
+    addEventListenerToElements('form.move-block', 'submit', (event: Event) =>
```

Comment:
just a qq- why the element called 'form.move-block' while in Line#10 it is just 'move-block'

Reply:
the other two are mistakes, thank you!